package-batches.el

```diff
@@ -0,0 +1,57 @@
+;;; package-batches.el --- Create batches for packages
```

Comment:
"batches" should be "badges", but anyway I think these things should be placed in a MELPA-specific file, e.g. `melpa.el`.

Reply:
Made the same in the descriptions of Kickstarter rewards. Of course there I was not allowed to change it and now I have to figure out how to send batches to users :wink:

Reply:
So I should just rename the file (but leave it in this repository)and reword some things a bit?

Reply:
> now I have to figure out how to send batches to users

For some of the rewards, you were offering batches of badges, tbh.